app/controllers/campaigns_controller.rb

```diff
@@ -50,6 +50,30 @@ def associate_seller_distributor
     json_response(campaign_json)
   end
 
+  # POST /campaigns/:id/gift_card
```

Comment:
So the idea is that they'd hit this over and over? A bulk/battch version of this endpoint might be a good follow

Reply:
yeah i changed this endpoint to make it bulk. i had initially had that in the ticket, but i was stuck on some of the nested validation in rails for a bit